File: GOVI-Back/src/main/java/com/sncf/govi/controller/model/FichierGOVI.java
package com.sncf.govi.controller.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.web.multipart.MultipartFile;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FichierGOVI {
    private TypeFichierEnum typeFichier;
    private MultipartFile multipartFile;
}

File: GOVI-Front/src/app/model/FichierGOVI.ts
import { TypeFichierEnum } from './TypeFichierEnum';

export class FichierGOVI {
  typeFichier: TypeFichierEnum;
  fichier: FormData;
  constructor(typeFichier: TypeFichierEnum, fichier: FormData) {
    this.typeFichier = typeFichier;
    this.fichier = fichier;
  }
}
